test/extended/testdata/bindata.go

```diff
@@ -51457,13 +51457,11 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//
-//	data/
-//	  foo.txt
-//	  img/
-//	    a.png
-//	    b.png
-//
+//     data/
```

Comment:
@tjungblu @hasbro17 @DennisPeriquet Hello ✋🏽 

the `make-verify` ci job fails and I tried to remove this or add it, it is always the same !!!

any thoughts ?